spec/factories/contacts.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :contact do
+    email "MyString"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.